DigitalEcosystem/exploratory/initial_work/initial_bg_work_commentated/bg_extra_descriptors.py

```diff
@@ -42,46 +59,71 @@
                       'is_bandgap_direct (unitless)',
                       'is_metal (unitless)',
                       'energy_vdw_per_atom (eV/atom)'])
-df
 
 
-# In[]:
+# # Featurize the Dataset
```

Comment:
(1) Do we ever explain where the dataset is coming from? If not, let's add a link here and/or in the README in the folder containing the datasets.
(2) let's spell out the names in full and include original naming for clarity, please - `2d_matpedia_dataset_raw.pkl`

Reply:
Added a readme in the raw data.

Changing the filename of this might break some of the exploratory notebooks, so I added a symlink with that name pointing to it.